Realm/RLMNetworkClient.mm

```diff
@@ -80,6 +78,14 @@ + (void)sendRequestToServer:(NSURL *)serverURL
                        JSON:(NSDictionary *)jsonDictionary
                     timeout:(NSTimeInterval)timeout
                  completion:(void (^)(NSError *, NSDictionary *))completionBlock {
+    // If the timeout isn't set then use the timeout set on the sync manager,
+    // or 60 seconds if it isn't set there either.
+    RLMSyncManager *syncManager = RLMSyncManager.sharedManager;
+    if (timeout < 1)
+        timeout = syncManager.timeoutOptions.connectTimeout / 1000.0;
+    if (timeout < 1)
```

Comment:
[nit] This might be slightly more legible if we used a ternary on the above line:
```objc
timeout = syncManager.timeoutOptions.connectTimeout / 1000.0 ?: 60.0;
```